pkg/codegen/schema/loader.go

```diff
@@ -152,6 +152,12 @@ func (l *pluginLoader) LoadPackageReference(pkg string, version *semver.Version)
 	if err != nil {
 		return nil, err
 	}
+	return l.loadPackageReferenceFromSchemaBytes(key, schemaBytes, version)
+}
+
+func (l *pluginLoader) loadPackageReferenceFromSchemaBytes(
```

Comment:
I ended up not needing this change can revert out.

Reply:
yah minimize diff if you can